### PR TITLE
wwan: fix extra bracket in some modem definitions

### PR DESCRIPTION
**Pull request by commodo**
Method used:
```
cd package/network/utils/wwan/files/data
for a in `ls` ; do sed -i $a 's/}}/}/g' ; done
for a in `ls` ; do sed -i $a -e 's/}\t"acm": 1/\t"acm": 1/g' ; done
for a in `ls` ; do sed -i $a -e 's/}\t"generic": 1/\t"generic": 1/g' ; done
```

Signed-off-by: Alexandru Ardelean <ardeleanalex@gmail.com>

### CODE REVIEW

**Comment by commodo** [comment]
@blogic

[notifying you, as maintainer ]

**Comment by lynxis** [comment]
@commodo it looks to me the json is still invalid.

File: package/network/utils/wwan/files/data/0421-060d [hunk 1]
```diff
@@ -2,5 +2,5 @@
 	"desc": "Nokia CS-10",
 	"control": 0,
 	"data": 0
```
Comment (lynxis) [inline package/network/utils/wwan/files/data/0421-060d]: e.g. missing colon here

**Comment by commodo** [comment]
urgs.....
my tunnel vision focused me on the double bracket only;

i'll use `for f in `ls` ; do echo $f ; python -m json.tool < $f || break ; done` to validate the JSON

maybe a travis job would be interesting for this stuff [ linting ]

**Comment by commodo** [comment]
@lynxis 
thanks for the catch

**Comment by commodo** [comment]
re-spinned and validated with `python -m json.tool`

**Comment by hauke** [comment]
Thank you, this was merged into LEDE master.

**Pull request closed without merging**